### ISSUE TITLE
HighlightJS ignores fenced code block style specifications

### ISSUE THREAD

**Issue by MicroTransactionsMatterToo** [pluralsight/guides-cms#102]
For some reason, Highlight JS stores codeblocks inside a `<pre>` block, which is given tag based on it's code syntax. However, for some reason, while the inner `<code>` block is correctly tagged with the language, the outer `<pre>` block ignores this, resulting in incorrectly highlighted code

**Comment by MicroTransactionsMatterToo** [comment]
Upon further investigation, it seems that it's a problem in which something keeps tagging `<pre>` blocks with `class="hljs sql"`. This is also part of the problem with the #100

**Comment by durden** [comment]
Good point.  Looks like it's an issue.  Lets make sure to test this on our new markdown rendering when it's ready.  I'll post back when that happens, hopefully this week.

**Comment by durden** [comment]
Can you test this on the site now?  We deployed our markdown rendering fixes today so hopefully this takes care of it.  Thanks!

**Comment by MicroTransactionsMatterToo** [comment]
Sure, will do
